recipes/ler/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ler" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ac75714d6d1d5da0c5726e6f07de984619e5b53282e32dc9a35fad4586a65fe6
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.10
+  run:
+    - astropy >=5.1
+    - bilby >=1.0.2
+    - gwsnr >=0.1
+    - lenstronomy >=1.10.4
+    - numba >=0.57.1
+    - numpy >=1.18
+    - pointpats >=2.3
+    - python
```

Comment:
You will need to change this to `python >=3.10` to make sure the runtime dependency is correct.